group-coordinator/src/main/java/org/apache/kafka/coordinator/group/assignor/OptimizedUniformAssignmentBuilder.java

```diff
@@ -59,7 +59,7 @@ public class OptimizedUniformAssignmentBuilder extends AbstractUniformAssignment
     /**
      * The assignment specification which includes member metadata.
      */
-    private final AssignmentSpec assignmentSpec;
+    private final GroupSpecImpl groupSpec;
```

Comment:
There are several places where we use GroupSpecImpl as the object type instead of GroupSpec. can we change all of them?

Reply:
thanks for the catch, I completely missed this!